command/kv/get/kv_get_test.go

```diff
@@ -418,3 +419,115 @@ func TestKVGetCommand_DetailedBase64(t *testing.T) {
 		t.Fatalf("bad %#v, value is not base64 encoded", output)
 	}
 }
+
+func TestKVGetCommand_KeysRecurse(t *testing.T) {
+	if testing.Short() {
+		t.Skip("too slow for testing.Short")
+	}
+
+	t.Parallel()
+	a := agent.NewTestAgent(t, ``)
+	defer a.Shutdown()
+	client := a.Client()
+
+	ui := cli.NewMockUi()
+	c := New(ui)
+	keys := map[string]string{
+		"foo/":   "",
+		"foo/a":  "Hello World 2",
+		"foo1/a": "Hello World 1",
+	}
+	for k, v := range keys {
+		var pair *api.KVPair
+		switch k {
```

Comment:
Did you want this to be the value?
```suggestion
		switch v {
```